<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServiceFeeWidget</class>
 <widget class="QWidget" name="ServiceFeeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>346</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>596</width>
    <height>346</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>596</width>
    <height>346</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ServiceFeeWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>8</number>
   </property>
   <property name="topMargin">
    <number>8</number>
   </property>
   <property name="rightMargin">
    <number>8</number>
   </property>
   <property name="bottomMargin">
    <number>8</number>
   </property>
   <item>
    <widget class="QWidget" name="ServiceFilletWidget" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#ServiceFilletWidget{background-color: rgb(255, 255, 255);border-radius:8px;}
</string>
     </property>
     <widget class="QLabel" name="labelIcon">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>110</width>
        <height>106</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>110</width>
        <height>106</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>110</width>
        <height>106</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/ZcloudMsgPopup/image/fee_tip.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="labelTilte">
      <property name="geometry">
       <rect>
        <x>189</x>
        <y>59</y>
        <width>350</width>
        <height>19</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>350</width>
        <height>19</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>19</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 18px &quot;微软雅黑&quot;;
color:#333333;</string>
      </property>
      <property name="text">
       <string>服务费到期提醒</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelContent">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>106</y>
        <width>350</width>
        <height>134</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>350</width>
        <height>134</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>134</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font:14px &quot;微软雅黑&quot;;
color:#666666;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QWidget" name="MsgBtnWidget" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>265</y>
        <width>560</width>
        <height>58</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>560</width>
        <height>58</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>560</width>
        <height>58</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ZcloudMsgPopup.qrc"/>
 </resources>
 <connections/>
</ui>
